_includes/post-meta-authors.html

```diff
@@ -0,0 +1,28 @@
+<div class="post-meta__author-list">
+  {% if post %}
+    {{ site.data.language.author_list }}
+  {% endif %}
+
+  {% if page.collection.size and page.themes.size %}
+    {% assign post = page %}
+  {% endif %}
+
+  {% if post.authors.size == 2 %}
+    {%- assign list_break = ' and ' -%}
+  {% elsif post.authors.size >= 3 %}
+    {%- assign list_break = ', ' -%}
+    {%- assign end_break = ' and ' -%}
+  {% endif %}
+
+    {%- for author in post.authors limit:3 -%}
```

Comment:
Last week, I wrote this limit in because I wasn't sure how to display all the authors as well as filter when the 'and' should be added into the array, because you can't loop backwards through an array. To fix this, I added in a pretty clunky if loop, that manually adds in the `end_break` depending on the number of authors. It's really clunky, but it works? I don't know if there's a better solution.